source/uri-options/uri-options.rst

```diff
@@ -280,6 +284,11 @@ pertaining to URI options apply here.
      - no
      - Relax TLS constraints as much as possible (e.g. allowing invalid certificates or hostname mismatches); drivers must document the exact constraints which are relaxed by this option being true
 
+   * - tlsDisableOcspEndpointCheck
+     - "true" or "false"
+     - Non-stapled OCSP is enabled
```

Comment:
In other places, we've linked directly to the other specs in this table; I think that makes sense here too

Reply:
I've added two links to this section :) One that explains what "if needed" means, and one that links to the more detailed explanation of the URI option